src/protocol.h

```diff
@@ -431,12 +431,20 @@ class CInv
     std::string ToString() const;
 
     // Single-message helper methods
-    bool IsMsgTx()        const { return type == MSG_TX; }
-    bool IsMsgWtx()       const { return type == MSG_WTX; }
+    bool IsUndefined() const { return type == UNDEFINED; }
+    bool IsMsgTx() const { return type == MSG_TX; }
+    bool IsMsgBlk() const { return type == MSG_BLOCK; }
```

Comment:
nit: maybe use "Block" to be in line with the surrounding code, e.g. `LookupBlockIndex()`, `MSG_BLOCK` which uses the full word. "Blk" looks like "bulk" to me rather than "block".

Reply:
Yes, I thought about this, but that would make the methods names quite a bit longer, and ISTM "blk" is an established abbreviation for "block", e.g. the filenames in ~/.bitcoin/blocks, and in the codebase `git grep -i blk`. Let's see what other reviewers say before changing (that change is actually in #19610, which needs to go in before this PR).